package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -47,9 +47,11 @@ supersede these two citations.)
    and O. Beckstein. `MDAnalysis: A Python package for the rapid analysis of
    molecular dynamics simulations`_. In S. Benthall and S. Rostrup, editors,
    *Proceedings of the 15th Python in Science Conference*, pages 98-105,
-   Austin, TX, 2016. SciPy.
+   Austin, TX, 2016. SciPy. doi:`10.25080/Majora-629e541a-00e`_
```

Comment:
I was given the DOI with this capitalization. However, duecredit lowercases it as a cite key so this is why it appears in lower case elsewhere.

The DOI resolver does not care about capitalization so either https://doi.org/10.25080/Majora-629e541a-00e or https://doi.org/10.25080/majora-629e541a-00e work just fine.

(If you follow the links you will see that the SSL certificate of the SciPy site is expired but that is beyond our control and is not changed if I use http instead of https.)